### ISSUE TITLE
Set up a system for people to review and test the tree working 

### ISSUE THREAD

**Issue by jrosindell** [OneZoom/OZtree#89]
1.) The One Tree, One Planet html page should include minimal customisation of its own - most of the changes should be to the main codebase as options that improve or don't change the behaviour of the existing pages.

**Comment by jrosindell** [comment]
2.) There is a question over the optimal method for sharing updated versions online for testing.  Use of the min life facility to have static pages pointed at the live server for data seems the best option currently on the table

**Comment by hyanwong** [comment]
This is currently set up using `grunt partial-local-install`. Anyone making a new testing version available for other people who do not run a full OneZoom installation can produce a minlife.html version that should work from a local file.

For this, the person making the minlife file needs to be running a local web2py instance, and also have `pics_dir=http://images.onezoom.org` set in appconfig.ini. They can then run `grunt partial-local-install` which will create a file in `static/minlife.html`. If they then  `git add static/minlife.html` and commit it to their local branch. 

Others can then pull this branch and should be able to `grunt compile` then simply open the `static/minlife.html` file as a file, without running a web server at all.

It may be a little more complex to e.g. test this on mobile.

**Comment by lentinj** [comment]
I've expanded ``partial-local-install`` a bit so it can be used for otop.html, and rsync'ed the static directory to my own server. Should be good for testing.

So refreshes keep working, I've added the following nginx rewrites:

```
  location /OZtree/static/ {
    rewrite ^(.*/\w+.html)/@.* $1;
    rewrite ^(.*)/\w+.html/(.*) $1/$2;
  }
```

The former strips off the node from the main URI, the latter bodges the now-incorrect relative URLs for images, etc. There's more efficient fixes but I don't think we care.
